server/serializer/channel_subscription_test.go

```diff
@@ -0,0 +1,180 @@
+	}{
+		"space subscription": {
+			subscription: []Subscription{
+				&SpaceSubscription{
+					SpaceKey: "TS",
+					BaseSubscription: BaseSubscription{
+						Alias:     "test",
+						BaseURL:   "https://test.com",
+						ChannelID: "testChannelID",
+						Events:    []string{CommentRemovedEvent, CommentUpdatedEvent},
+					},
+				},
+			},
+			result: "#### Space Subscriptions \n| Name | Base URL | Space Key | Channel ID | Events|\n| :----|:--------| :--------| :-----|\n|test|https://test.com|TS|testChannelID|Comment Remove, Comment Update|",
+		},
+		"page subscription": {
+			subscription: []Subscription{
+				&PageSubscription{
+					PageID: "12345",
+					BaseSubscription: BaseSubscription{
+						Alias:     "abc",
+						BaseURL:   "https://test.com",
+						ChannelID: "testChannelID",
+						Events:    []string{CommentCreatedEvent, CommentUpdatedEvent},
+					},
+				},
+			},
+			result: "#### Page Subscriptions \n| Name | Base URL | Page ID | Channel ID | Events|\n| :----|:--------| :--------| :-----|\n|abc|https://test.com|12345|testChannelID|Comment Create, Comment Update|",
+		},
+	}
+
+	for name, tc := range tests {
+		t.Run(name, func(t *testing.T) {
+			defer monkey.UnpatchAll()
+
+			formattedList := FormattedOldSubscriptionList(tc.subscription)
+			assert.Equal(t, tc.result, formattedList)
+		})
+	}
+}
+
+func TestGetOldFormattedSubscription(t *testing.T) {
+	tests := map[string]struct {
+		subscription Subscription
+		result       string
+	}{
+		"space subscription": {
+			subscription: &SpaceSubscription{
+				SpaceKey: "TS",
+				BaseSubscription: BaseSubscription{
+					Alias:     "test",
+					BaseURL:   "https://test.com",
+					ChannelID: "testChannelID",
+					Events:    []string{CommentRemovedEvent, CommentUpdatedEvent},
+				},
+			},
+			result: "\n|test|https://test.com|TS|testChannelID|Comment Remove, Comment Update|",
+		},
+		"page subscription": {
+			subscription: &PageSubscription{
+				PageID: "12345",
+				BaseSubscription: BaseSubscription{
+					Alias:     "abc",
+					BaseURL:   "https://test.com",
+					ChannelID: "testChannelID",
+					Events:    []string{CommentCreatedEvent, CommentUpdatedEvent},
+				},
+			},
+			result: "\n|abc|https://test.com|12345|testChannelID|Comment Create, Comment Update|",
+		},
+	}
+
+	for name, tc := range tests {
+		t.Run(name, func(t *testing.T) {
+			defer monkey.UnpatchAll()
+
+			formattedSubscription := tc.subscription.GetOldFormattedSubscription()
+			assert.Equal(t, tc.result, formattedSubscription)
+		})
+	}
+}
+
+func TestGetFormattedSubscription(t *testing.T) {
+	tests := map[string]struct {
+		subscription Subscription
+		result       string
+	}{
+		"space subscription": {
+			subscription: &SpaceSubscription{
+				SpaceKey: "TS",
+				BaseSubscription: BaseSubscription{
+					Alias:     "test",
+					BaseURL:   "https://test.com",
+					ChannelID: "testChannelID",
+					Events:    []string{CommentRemovedEvent, CommentUpdatedEvent},
+				},
+			},
```

Comment:
we can have util to get space and page subscription, as they have same values for all test cases